packages/playground/src/portal/twin_view.vue

```diff
@@ -52,16 +63,38 @@
 import { generatePublicKey } from "@threefold/rmb_direct_client";
 import { onMounted, ref } from "vue";
 
+import router from "../router";
 import { useProfileManager } from "../stores";
+import type { Farm } from "../types";
+import { getFarms } from "../utils/get_farms";
 import { getGrid } from "../utils/grid";
-
 const profileManager = useProfileManager();
 
 const editingTwin = ref(false);
 const relay = ref(profileManager.profile?.relay || "");
 const updateRelay = ref(false);
 const errorMsg = ref("");
+const openVotePopup = ref(false);
+const numberOfActiveProposals = ref(0);
+const userFarms = ref<Farm[]>();
+onMounted(async () => {
+  const profile = profileManager.profile!;
+  const grid = await getGrid(profile);
+  if (grid) {
+    userFarms.value = await getFarms(grid, { ownedBy: profile.twinId }, {});
+    if (userFarms.value.length > 0) {
+      const proposals = grid?.dao.get();
+
+      const activeProposals = (await proposals)?.active;
+      numberOfActiveProposals.value = activeProposals ? activeProposals.length : 0;
 
+      openVotePopup.value = true;
```

Comment:
we should check if the active proposal length is greater than 0 to show the pop-up

Reply:
also, we should check if the user already voted on the proposal. And if yes the pop-up shouldn't be shown.